index.html

```diff
@@ -49,5 +49,169 @@ <h1 class="h1">I'm Ezechukwu moses Glad to see you!</h1>
+        <div class="card-contents"></div>
+        <h2>Mult-Post Stories</h2>
+        <ul class="works">
+          <li>Canopy</li>
+          <li><img src="./image/Counter.png" alt="Counter" /></li>
+          <li>Back End Dev</li>
+          <li><img src="./image/Counter.png" alt="Counter" /></li>
+          <li>2015</li>
+        </ul>
+        <p class="p">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul >
+          <ul class="works2">
+          <li class="work2-li"><img src="./image/Tag.png" alt="html" /></li>
+          <li><img src="./image/Tag1.png" alt="css" /></li>
+          <li><img src="./image/Tag2.png" alt="javaScript" /></li>
+        </ul>
+        <div class="div">
+          <button class="btn-big btn">see project</button>
+        </div>
+      </div>
+      
+
+      <!-- third work section -->
+
+      <div class="card">
+        <img class="img" src="./image/Snapshoot3.png" alt="ID" />
+        <div class="card-contents"></div>
+        <h2>Tonic</h2>
+        <ul class="works">
+          <li>CANOPY</li>
+          <li><img src="./image/Counter.png" alt="Counter" /></li>
+          <li>Back End Dev</li>
+          <li><img src="./image/Counter.png" alt="Counter" /></li>
+          <li>2015</li>
+        </ul>
+        <p class="p">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="works2">
+          <li><img src="./image/Tag.png" alt="html" /></li>
+          <li><img src="./image/Tag1.png" alt="css" /></li>
+          <li><img src="./image/Tag2.png" alt="javaScript" /></li>
+        </ul>
+        <div class="div">
+          <button class="btn-big btn">see project</button>
+        </div>
+      </div>
+
+      <!-- fourth work section -->
+
+      <div class="card">
+        <img class="img" src="./image/Snapshoot4.png" alt="ID" />
+        <div class="card-contents"></div>
+        <h2>Mult-Post Stories</h2>
+        <ul class="works">
+          <li>CANOPY</li>
+          <li><img src="./image/Counter.png" alt="Counter" /></li>
+          <li>Back End Dev</li>
+          <li><img src="./image/Counter.png" alt="Counter" /></li>
+          <li>2015</li>
+        </ul>
+        <p class="p">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="works2">
+          <li><img src="./image/Tag.png" alt="html" /></li>
+          <li><img src="./image/Tag1.png" alt="css" /></li>
+          <li><img src="./image/Tag2.png" alt="javaScript" /></li>
+        </ul>
+        <div class="div">
+          <button class="btn-big btn">see project</button>
+        </div>
+      </div>
+    </section>
+    
+    <div class="bg-white">
+      <section id="aboutme">
+        <div class="me">
+          <h3 class="aboutme">About Myself</h3>
+          <p class="p">
```

Comment:
- Hey, I see you've used one class to style all the paragraph which is a good idea, however, do bear in mind that you're styling on the paragraphs are not consistent comparing it with the Figma design. see the image below:
   - kindly reduce the font weight of all the paragraphs in each card, about the section, and the first section.
   - use the correct color and font size on the `LET'S CONNECT`



*YOUR'S*
![Screenshot from 2023-01-13 23-18-35](https://user-images.githubusercontent.com/59397159/212430315-cf4e1586-10d1-44ae-bcfd-ae02c7b4d2ff.png)


*YOUR'S*
![Screenshot from 2023-01-13 23-20-48](https://user-images.githubusercontent.com/59397159/212430287-6c081af7-df91-4f4c-af5d-87b91588f3fb.png)

*FIGMA*
![Screenshot from 2023-01-13 23-20-21](https://user-images.githubusercontent.com/59397159/212430298-878b4563-9c0f-4b8d-ab61-02c12727083a.png)

*FIGMA*
![Screenshot from 2023-01-13 23-19-56](https://user-images.githubusercontent.com/59397159/212430306-994afbbd-6dec-4ab1-ae87-ee2012331911.png)

Reply:
### README FILE


- Hey, please, let's update our readme using this [template](https://github.com/microverseinc/readme-template) to conform to the current readme version as yours is not the updated version.

Reply:
- [ ] You have really deployed great effort to reconcile the differences between your design and the requirement as required earlier, and you are just left with a few more details to look into 💚 :
- Kindly check for preferred `values` for your `padding`, `margin`, `font-size` , `font-weight` or `color`. This will save you stress 🙏 
- Please check the look of your `about me` 
![image](https://user-images.githubusercontent.com/99715640/212444574-3b59a537-6391-45d1-ba96-8d17010d3430.png)

- See what Figma looks like:
![image](https://user-images.githubusercontent.com/99715640/212444609-12cd7ac6-2909-410c-8f01-a78ff70f58cc.png)

Reply:
Thanks for the review @AbayomiOlaoye